modules/virt-storage-wizard-fields-web.adoc

```diff
@@ -26,3 +26,36 @@
 |The `StorageClass` that is used to create the disk.
 
 |===
+
+[id="virt-storage-wizard-fields-advanced-web_{context}"]
+[discrete]
+== Advanced storage settings
+The following advanced storage settings are available for *Blank*, *URL*, and *Attach Cloned Disk* disks.
+These parameters are optional. If you do not specify these parameters, the defaults are used, based on what has been configured in the `kubevirt-storage-class-defaults` ConfigMap.
+
+[cols="2a,3a,5a",caption=]
+.Advanced storage settings
+|===
+|Name | Parameter |  Description
+
+.2+|Volume Mode
+|Filesystem
+|Creates a filesystem on the virtual disk.
+
+|Block
+|Creates a block disk. Only use `Block` if the underlying storage supports it.
```

Comment:
Stores the virtual disk directly on the block volume.  Only use `Block` if the underlying storage supports it.